### PR TITLE
Set default slack channel value to #general

### PR DESCRIPTION
**Pull request by ananichev**
This PR:
- adds default slack channel value to #general https://github.com/Calzada-Code/remote-coffee-slack/issues/9

### CODE REVIEW

File: lib/remote_coffee_slack.rb [hunk 1]
```diff
@@ -28,6 +28,13 @@ def self.notify_next_session
   end
 
   class Configuration
-    attr_accessor :slack_bot_token, :slack_channel, :members_per_group
+    DEFAULT_SLACK_CHANNEL = '#general'.freeze
+
+    attr_accessor :slack_bot_token, :members_per_group
+    attr_writer :slack_channel
```
Comment (JuanCrg90) [inline lib/remote_coffee_slack.rb]: Could you move this back to `attr_accessor` please?

File: test/remote_coffee_slack_test.rb [hunk 2]
```diff
@@ -1,4 +1,8 @@
 require "test_helper"
 
 class RemoteCoffeeSlackTest < Minitest::Test
+  def test_default_configuration
```
Comment (JuanCrg90) [inline test/remote_coffee_slack_test.rb:4]: could you add an extra test to ensure if you set the channel this is updated?

**Review by JuanCrg90 — CHANGES_REQUESTED** [review]
I have added some feedback.

Thanks in advance :D

**Comment by ananichev** [comment]
@JuanCrg90 done

File: test/remote_coffee_slack_test.rb [hunk 3]
```diff
@@ -1,4 +1,14 @@
...
+    config = RemoteCoffeeSlack::Configuration.new
+    assert_equal config.slack_channel, '#general'
```
Comment (JuanCrg90) [inline test/remote_coffee_slack_test.rb]:
```suggestion
    assert_equal  '#general', config.slack_channel
```

File: test/remote_coffee_slack_test.rb [hunk 4]
```diff
@@ -1,4 +1,14 @@
...
+  end
+
+  def test_configuration
+    config = RemoteCoffeeSlack::Configuration.new
+    config.slack_channel = '#new_channel'
+    assert_equal config.slack_channel, '#new_channel'
```
Comment (JuanCrg90) [inline test/remote_coffee_slack_test.rb]:
```suggestion
    assert_equal '#new_channel', config.slack_channel
```

**Review by JuanCrg90 — CHANGES_REQUESTED** [review]
Minitest `assert_equal` syntax requires expectation as the first parameter.

http://docs.seattlerb.org/minitest/Minitest/Assertions.html
> assert_equal(exp, act, msg = nil)

BTW could you update the docs mentioning this configurable parameter?

File: test/remote_coffee_slack/members_test.rb [hunk 5]
```diff
@@ -1,7 +1,8 @@
 require "test_helper"
 
-class RemoteCoffeeSlack::MembersTest  < Minitest::Test
+class RemoteCoffeeSlack::MembersTest < Minitest::Test
   def setup
+    RemoteCoffeeSlack.config.members_per_group = 2
```
Comment (JuanCrg90) [inline test/remote_coffee_slack/members_test.rb]: Why did you add this line? The default value must be 2, shouldn't be necessary to set a default parameter.

Comment (ananichev) [reply test/remote_coffee_slack/members_test.rb]: This test is a flaky one.
Sometimes this value is 4 and it fails

File: test/remote_coffee_slack/members_test.rb [hunk 6]
```diff
@@ -17,6 +17,7 @@ def test_select_coffee_mates_with_members
       RemoteCoffeeSlack.config.members_per_group = 4
```
Comment (ananichev) [inline test/remote_coffee_slack/members_test.rb:17]: @JuanCrg90  looks like this test did not update configuration back to 2 members per group.
That's why the previous one was failing when run after this one

Comment (JuanCrg90) [reply test/remote_coffee_slack/members_test.rb:17]: Could you open an issue for this please?

Comment (ananichev) [reply test/remote_coffee_slack/members_test.rb:17]: I fixed it here https://github.com/Calzada-Code/remote-coffee-slack/pull/13/commits/da63629dc39e60a02b580fed0ff751ba2d7715a3 (my last commit to this PR)

**Comment by ananichev** [comment]
Yeah, you are right, it should be the default value

**Review by JuanCrg90 — APPROVED** [review]
